examples/dojo/package.json

```diff
@@ -1,4 +1,13 @@
 {
+	"private": true,
+	"dependencies": {
+		"todomvc-app-css": "^1.0.0",
```

Comment:
package.json uses and will always use 2 spaces. [That's stated by our editorconfig too](https://github.com/tastejs/todomvc/commit/64484b9e18eabed01b8314af899f50697d167946).